### ISSUE TITLE
hello, how to use the esp32-cam  ?

### ISSUE THREAD

**Issue by kyrosle** [wouterdebie/esp32-cam-rs#1]
i have tried but it occurs error in the cmake stage

**Comment by wouterdebie** [comment]
Yes, this is not a working thing. I merely put it up to refer to it in some issues I opened in esp-idf-sys and others.

**Comment by kyrosle** [comment]
did you see this code project?:  https://github.com/ChenHuaYou/esp-cam-rs/blob/main/src/main.rs

**Comment by kyrosle** [comment]
This project has reference value: https://github.com/kyrosle/esp32cam

**Comment by wouterdebie** [comment]
Thanks! I had seen the first one.
